web_src/less/_base.less

```diff
@@ -181,6 +181,7 @@ details summary > * {
 
 ::-webkit-scrollbar {
   width: 10px;
+  height: 10px;
```

Comment:
I would assume that all scrollbars are now quadratic because of this.

Or does the scrollbar internally ignore the non-relevant setting?

Reply:
Why don't you try it?

Reply:
Am currently trying it.
Can you please point me somewhere where the horizontal scrollbars should appear?
Because right now I don't see any horizontal scrollbars on the admin page in Chrome on a minimum width window…

Reply:
Does Chromium behave differently than Chrome here?
Because for me, the admin page looks like this:
![image](https://user-images.githubusercontent.com/51889757/133921764-82480761-211a-4f89-876c-567cf22714e3.png)

I thought Chrome and Chromium were basically the same thing, with the only difference being that Chromium gets rid of most Google things.

Reply:
Ah, I see now:
For some reason up until about the half screen, this scrollbar will be displayed.
Only when the screen gets smaller, the tabs will be wrapped.
I would prefer unified behavior:
Either always wrap and never display the scrollbar, **or** never wrap and always use the scrollbar.
But that is most likely worth its own PR, so for now: lgtm.